doc/tutorial.txt

```diff
@@ -422,6 +422,39 @@ following sequence:
 	"unlock" : "mutexA"
 }
 
+* barrier : String. Used as at least a pair where the name must match.
```

Comment:
Great! Maybe add some comments about this in the documentation so that the behaviour is clear?